src/include/execution/tpl.h

```diff
@@ -0,0 +1,4 @@
+#pragma once
```

Comment:
I removed the configure_file step that used tpl.h.in to build this file at configure time.